crates/database/src/orders.rs

```diff
@@ -567,6 +579,26 @@ pub async fn single_full_order(
     sqlx::query_as(QUERY).bind(uid).fetch_optional(ex).await
 }
 
+pub async fn single_full_order_with_quote(
```

Comment:
Create a new query so we don't need to do two roundtrip to get an order with its quote data.